### DISCUSSION TITLE
Python version

### DISCUSSION THREAD

**Discussion by ptabandzelic** [databrickslabs/dbx#659] [Q&A]
Hey, i have my python project for databricks, i am using DBR runtime version 11.3 which uses python 3.9. Should i use then strict python version of 3.9 for my project because DBR uses 3.9, or it is ok if i just python python3.8 and higher.

**Comment by copdips** [comment]
well, databricks community is a better place to raise such question.
Each databricks runtime has a specific python version installed, it's better to use the same version.
and during the CI, you can test your code with current and higher Python version for future migration, but not lower Python version.
